### DISCUSSION TITLE
GitHub Pages not working properly

### DISCUSSION THREAD

**Discussion by slf188** [community/community#23692] [Other Feature Feedback, Questions, & Ideas]
<p>Hey guys, I wanted to host my website on GitHub pages but after many attemps my website seems to be not working properly. Here is the link of my repo <a href="https://github.com/slf188/vallejo.ai" rel="noopener nofollow ugc">https://github.com/slf188/vallejo.ai</a>[My repo]. Please help!</p>

**Comment by kingthorin** [comment, marked as answer]
<p>What you’ve built is here:</p>
<ul>
<li><a href="https://slf188.github.io/vallejo.ai/" rel="noopener nofollow ugc">https://slf188.github.io/vallejo.ai/</a></li>
<li><a href="https://slf188.github.io/vallejo.ai/felipe.ai.html" rel="noopener nofollow ugc">https://slf188.github.io/vallejo.ai/felipe.ai.html</a></li>
</ul>

**Comment by slf188** [comment]
<p>Thank you so much, now it finally worked out. Appreciate it!</p>
